query-planner-wasm/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "query-planner-wasm" # this ends up being the name in the published npm package.
-version = "0.0.4"
+version = "0.0.0-DO.NOT.CHANGE" # The version is managed in the package.json, by Lerna.
```

Comment:
(apologies for the late reply, I didn't get to look at this yesterday, it was a short day for me that was packed with meetings in the morning)

Is there any way we can keep these in sync instead of forcing this version to be a stub? 
If possible, I don't want to prevent us from being able to publish this crate as a standalone to crates.io

Reply:
This doesn't prevent us from doing it permanently, only in the current iteration of the natural growth of this project.